### PR TITLE
drf-nested-routers: update to version 0.92

### PR DESCRIPTION
**Pull request by peter-stadler**
Maintainer: me
Compile tested: x86_64, x86_64 qemu, master
Run tested: x86_64, x86_64 qemu, master, run etesync-server

Description: update to newest version <strike>using github until pypi has right file</strike> (*edit*: it is fixed).
Changes: https://github.com/alanjds/drf-nested-routers/compare/v0.91.0...v0.92.1

### CODE REVIEW

File: lang/python/python3-drf-nested-routers/Makefile [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
 PKG_LICENSE:=Apache-2.0
 PKG_LICENSE_FILES:=LICENSE
 PKG_MAINTAINER:=Peter Stadler <peter.stadler@student.uibk.ac.at>
 
-include ../pypi.mk
+#include ../pypi.mk ### not working, use github instead:
+PKG_SOURCE:=v$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/alanjds/drf-nested-routers/archive/
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
+HOST_BUILD_DIR:=$(BUILD_DIR_HOST)/$(PKG_NAME)$-$(PKG_VERSION)
```
Comment (jefferyto) [inline lang/python/python3-drf-nested-routers/Makefile:18]: There is no host build of this package so I don't think `HOST_BUILD_DIR` is necessary.

**Review by jefferyto — COMMENTED** [review]
Perhaps you can open an issue [upstream](https://github.com/alanjds/drf-nested-routers) to let them know about the broken source tarball on PyPI?

File: lang/python/python3-drf-nested-routers/Makefile (hunk 1 above)
Comment (peter-stadler) [reply lang/python/python3-drf-nested-routers/Makefile:18]: Thx, I will remove it (I copied it from pypi.mk)

**Comment by peter-stadler** [comment]
I will open an issue upstream, thank you :-)

**Comment by peter-stadler** [comment]
Using upstream fix (and rebased).

File: lang/python/python3-drf-nested-routers/Makefile [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
 PYPI_NAME:=drf-nested-routers
-PKG_HASH:=46e5c3abc15c782cafafd7d75028e8f9121bbc6228e3599bbb48a3daa4585034
+PKG_HASH:=e043fc937f94ac462a92d2d9fc9a7e55710a67164b558442adfe9634fc519c3b
 
 PKG_LICENSE:=Apache-2.0
 PKG_LICENSE_FILES:=LICENSE
 PKG_MAINTAINER:=Peter Stadler <peter.stadler@student.uibk.ac.at>
 
 include ../pypi.mk
+
```
Comment (BKPepe) [inline lang/python/python3-drf-nested-routers/Makefile:15]: why?

Comment (peter-stadler) [reply lang/python/python3-drf-nested-routers/Makefile]: I forgot to remove this after going back to the pypi version, corrected it now. Thank you :-)

**Pull request merged**